src/modelfree/configs/multi/score.py

```diff
@@ -111,6 +112,15 @@ def high_accuracy(score):
         score['episodes'] = 1000
         score['num_env'] = 16
 
+    @multi_score_ex.named_config
+    def medium_accuracy(score):
+        score = dict(score)
+        score['episodes'] = 100
+        score['num_env'] = 16
+        exp_name = 'medium_accuracy_'
```

Comment:
Adding a tag seems good. But won't this just override the experiment name? I think this needs to be `'medium_acuracy_' + exp_name`. For consistency should probably also update `high_accuracy` to add a tag similarly.

Reply:
My recollection was that exp_name wasn't defined at this point because of the order of how named_configs were run, but I can go back and try to confirm that.

Reply:
Yeah, I guess; it seems in general worryingly fragile to have configs that only work if you run them in a certain order we'd have to document somewhere. It seems like the most sensible thing to do would be to have the base config be something like an empty string, and then add to it with each named config, but that gets broken by Sacred's  decision about the order of running named vs not-named configs.

Reply:
I agree this is not ideal. However, the current implementation is also sensitive to the order in which you run things. If you do `medium_accuracy` last, it will clobber the experiment name. Better to at least be consistent with the existing code.

There might be some way of detecting if exp_name was already set? Maybe by a default parameter?